boston_housing/project/mlcube.py

```diff
@@ -0,0 +1,73 @@
+"""MLCube handler file"""
```

Comment:
This is a great main file - and I'm thinking about how we can further simplify this in the spec; nothing need to change here, but long term we should think about how to make this file simple for people to create :)